internal/ls/format.go

```diff
@@ -137,6 +137,9 @@ func getRangeOfEnclosingComment(
 	precedingToken *ast.Node,
 	tokenAtPosition *ast.Node,
 ) *ast.CommentRange {
+	if tokenAtPosition == nil {
+		tokenAtPosition = astnav.GetTokenAtPosition(file, position)
+	}
```

Comment:
Pretty sure @gabritto omitted this intentionally in a recent PR (might be mixing details up).

The comment above says 

```
// Unlike the TS implementation, this function *will not* compute default values for
// `precedingToken` and `tokenAtPosition`.
```

so I think you should explicitly pass in a `tokenAtPosition`.

Reply:
Ah yes, thanks for catching that, I didn't realize this PR was using this function.